### ISSUE TITLE
budo and heroku

### ISSUE THREAD

**Issue by rimager** [mattdesl/budo#174]
Can I deploy budo to heroku? 

Here is my package.json
 
`  "scripts": {
    "start": "budo index.js --serve=bundle.js",
    "test": "eslint --no-eslintrc -c .eslintrc index.js",
    "build" : "browserify index.js | uglifyjs -c -m > bundle.js"
  },`

**Comment by pinn3** [comment]
Yes you can, but it's not recommended as budo is a **development** server, not intended for **production**. It watches and builds your files, then serves the built bundle. Whenever you deploy your app to Heroku, the service will restart anyways and therefor the watch functionality wont serve any purpose.

Also, if you only want to serve a static page, then Heroku might be a bit overkill. I suggest that you build the the bundle with your build-script, and then deploy it on a static web server, such as [surge](http://surge.sh/).
